democracy_club/templates/about/index.html

```diff
@@ -54,37 +47,19 @@
 by an online network of hundreds of volunteers who assist with compiling and
 maintaining our databases.
 
-Much of our work focuses on collecting data from the UK's approximately 400 local
-authorities, either via communicating with them directly or taking information
-from their websites using our unique infrastructure of web 'scrapers'. Working with
-our amazing volunteers, we aggregate this data and publish it for use by anyone.
-We are committed to open data. We believe that making the data that we collect
-available to everyone is the best way of empowering the most people to vote.
+Much of our work involves collecting data from the UK's approximately 350 local authorities, either by communicating with them directly or taking information from their websites using our unique infrastructure of automated 'scrapers'. Working with our amazing volunteers, we aggregate this data and publish it for use by anyone. 
```

Comment:
I love a good scraper, but I wonder if talking about scrapers is the right pitch for the audience of this website? Feels like it is really "in the weeds" stuff when we should be talking in more broad brush strokes.

Reply:
What would be your alternative? Simply 'automation'?

Reply:
Maybe it doesn't matter. Maybe Sym has thoughts on this one. If not, lets not block on it.